static/app/components/searchSyntax/parser.tsx

```diff
@@ -302,6 +306,13 @@ type TokenConverterOpts = {
   text: TextFn;
 };
 
+type AggregateFilter =
+  | FilterType.AggregateDate
+  | FilterType.AggregateDuration
+  | FilterType.AggregateNumeric
+  | FilterType.AggregatePercentage
+  | FilterType.AggregateRelativeDate;
```

Comment:
Let's put this before `filterTypeConfig`, since it's closer to the `FilterType` definitions then